projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-detail-items/order-consigned-entries/order-consigned-entries.component.html

```diff
@@ -21,7 +21,7 @@
   </div>
   <div class="cx-list-item col-12">
     <cx-cart-item-list
-      [items]="getConsignmentProducts(consignment)"
```

Comment:
Is the ` getConsignmentProducts` still used?  Otherwise perhaps it should be deprecated.

Reply:
not used.